core/matching/matchers/matchers.go

```diff
@@ -18,4 +20,9 @@ var Matchers = map[string]MatcherFunc{
 	ContainsExactly: ContainsExactlyMatch,
 	ContainsOnly:    ContainsOnlyMatch,
 	Contains:        ContainsMatch,
+	Array:           ContainsExactlyMatch,
```

Comment:
Are you going to replace the other `Contains...` matchers with `Array`? so `Array` without config is the same as `ContainsExcactly` and if you want to do `Contains` then you pass `ignoreOrder: true, ignoreUnknown: true` as the config to the `Array` matcher.

Reply:
Yea, we will be deprecating it soon.